src/components/SearchDrawer/index.tsx

```diff
@@ -0,0 +1,184 @@
+import { FormattedMessage as Msg } from 'react-intl';
+import { useQuery } from 'react-query';
+import { useRouter } from 'next/router';
+import {
+    Box,
+    Container,
+    Typography,
+} from '@material-ui/core';
+import { FunctionComponent, useEffect, useRef, useState } from 'react';
+
+import getSearchDrawerResults from '../../fetching/getSearchDrawerResults';
+import useDebounce from '../../hooks/useDebounce';
+
+import ResultsList from './ResultsList';
+import SearchField from './SearchField';
+
+export const MINIMUM_CHARACTERS = 3;
+
+interface SearchDrawerProps {
+    orgId: string;
+}
+
+const SearchDrawer: FunctionComponent<SearchDrawerProps> = ({ orgId }): JSX.Element | null => {
+    const router = useRouter();
+
+    const drawer = useRef<HTMLDivElement>(null);
+    const [drawerOpen, setDrawerOpen] = useState(false);
```

Comment:
If possible, I think `setDrawerOpen(false)` should also blur the text field. I don't know how best to solve this so I'm putting this comment here and you can decide whether it's an easy fix or something that should be tracked in a separate issue for later.

Reply:
The only time that the compenent would not blur would be when pressing "escape". To fix this I passed a ref down to the input, and when pressing escape, blured the input. This also meant my `onClick` handler was no longer needed, so it was removed.